src/google/protobuf/compiler/java/java_message_field.cc

```diff
@@ -299,6 +300,7 @@ GenerateBuilderMembers(io::Printer* printer) const {
     "$name$Builder_.setMessage(value);\n",
 
     "$set_has_field_bit_builder$\n"
+    "// @@protoc_insertion_point(builder_field_modifier_scope_after:$containing_type_full_name$.set$capitalized_name$)\n"
```

Comment:
What about the insertion points in this file and the other below:  shall I try to rename them to something like @@protoc_insertion_point(builder_field_XXX:$full_name$) too?

Reply:
Done